lib/Constants.js

```diff
@@ -370,5 +370,241 @@ module.exports = {
   },
   commandClass (cmd) {
     return this._commandClassMap[cmd] || 'unknownClass_' + cmd
+  },
+  _genericDeviceClassMap: {
+    // https://github.com/home-assistant/core/blob/dev/homeassistant/components/zwave/const.py#L196
+    // 0x00: specific_type_not_used // Available in all Generic types
+    0x03: {
+      genericType: 'generic_type_av_control_point',
+      specificTypes: {
+        0x12: 'specific_type_doorbell',
+        0x04: 'specific_type_satellite_receiver',
+        0x11: 'specific_type_satellite_receiver_v2'
+      }
+    },
+    0x04: {
+      genericType: 'generic_type_display',
+      specificTypes: {
+        0x01: 'specific_type_simple_display'
+      }
+    },
+    0x40: {
+      genericType: 'generic_type_entry_control',
+      specificTypes: {
+        0x01: 'specific_type_door_lock',
+        0x02: 'specific_type_advanced_door_lock',
+        0x03: 'specific_type_secure_keypad_door_lock',
+        0x04: 'specific_type_secure_keypad_door_lock_deadbolt',
+        0x05: 'specific_type_secure_door',
+        0x06: 'specific_type_secure_gate',
+        0x07: 'specific_type_secure_barrier_addon',
+        0x08: 'specific_type_secure_barrier_open_only',
+        0x09: 'specific_type_secure_barrier_close_only',
+        0x10: 'specific_type_secure_lockbox',
+        0x11: 'specific_type_secure_keypad'
+      }
+    },
+    0x01: {
+      genericType: 'generic_type_generic_controller',
+      specificTypes: {
+        0x01: 'specific_type_portable_controller',
+        0x02: 'specific_type_portable_scene_controller',
+        0x03: 'specific_type_portable_installer_tool',
+        0x04: 'specific_type_control_av',
+        0x06: 'specific_type_control_simple'
+      }
+    },
+    0x31: {
+      genericType: 'generic_type_meter',
+      specificTypes: {
+        0x01: 'specific_type_simple_meter',
+        0x02: 'specific_type_adv_energy_control',
+        0x03: 'specific_type_whole_home_meter_simple'
+      }
+    },
+    0x30: {
+      genericType: 'generic_type_meter_pulse',
+      specificTypes: {},
+    },
+    0x0f: {
+      genericType: 'generic_type_non_interoperable',
+      specificTypes: {},
+    },
+    0x15: {
```

Comment:
`0x15` is repeated twice in this map

Reply:
Fixed in dd59a255189252f55124f938b40d0eb2cc52cfb1